scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/entrypoints/AutoBattle.kt

```diff
@@ -460,7 +461,10 @@ open class AutoBattle @Inject constructor(
             3.seconds.wait()
         } else if (waitAPRegenPrefs) {
             Location(1300, 1240).click();
-            20.seconds.wait()
+            for (i in 1..waitAPRegenMinutePrefs) {
```

Comment:
I think if you reverse the order, it should be fine:
```suggestion
            for (i in waitAPRegenMinutePrefs..1) {
                toast("Waiting for AP regeneration. Re-checking in $i minutes.")

Reply:
Oh and shouldn't this message be translatable?

Reply:
Right, it is! To be honest I'm not sure how to put the translatable string here, what should I do?

Reply:
Look into `IScriptMessages`. Similar code is used for showing AP running out.